### ISSUE TITLE
References for payments

### ISSUE THREAD

**Issue by ithustle** [flowck/proxypay-api#6]
Is it possible create a reference without expire date?

**Comment by ithustle** [comment]
finger

**Comment by firminochangani** [comment]
No, it's not possible. You may want to read more the official documentation from here: https://developer.proxypay.co.ao/v2/#create-or-update-reference
